src/main/kotlin/com/example/pokemon_api/model/Pokemon.kt

```diff
@@ -0,0 +1,12 @@
+package com.example.pokemon_api.model
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+data class Pokemon(
+    val id: Int,
+    val name: String,
+    val type: List<String>,
```

Comment:
```suggestion
    val types: List<String>,
```

It's a list after all, isn't it?

Reply:
Thats true, better semantics... done with @JsonProperty to keep the Json structure.

Reply:
So that works, until you call the endpoint, where it turns back to singular `type`.
Any chance we could address that as well?

Reply:
I updated it to use @JsonAlias("type") so the API now returns types in the response while still accepting type in the input.